src/zone.rs

```diff
@@ -40,7 +51,11 @@ impl Zone {
         }
     }
 
-    pub fn from_osm(relation: &Relation) -> Option<Self> {
+    pub fn set_parent(&mut self, idx: Option<ZoneIndex>) {
```

Comment:
do you think we need this ?

Reply:
Maybe not. Unless we want to encapsulate handling of symmetrical relations (eg: keep references from the parent to a list of children).